packages/gestalt/src/ModuleExpandable.js

```diff
@@ -23,9 +25,13 @@ export default function ModuleExpandable({
   id,
   accessibilityExpandLabel,
   accessibilityCollapseLabel,
+  expandedId,
+  setExpandedId,
   items,
 }: Props): Node {
-  const [expandedId, setExpandedId] = useState(-1);
+  const [localExpandedId, setLocalExpandedId] = useState(null);
```

Comment:
Let's keep this as it was, so we don't have to worry about internal and external state.